smarts/sstudio/genhistories.py

```diff
@@ -450,6 +460,180 @@ def column_val_in_row(self, row, col_name):
         return None
 
 
+class Waymo(_TrajectoryDataset):
+    def __init__(self, dataset_spec, output):
```

Comment:
Consider input/output type hinting for all the functions which we modify or write.